src/common/Notes/NoteCreate.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { Redirect } from 'react-router-dom';
+
+import { NoteCreatePage } from '@folio/stripes/smart-components';
+
+import {
+  ORDERS_DOMAIN,
+  ORDERS_ROUTE,
+} from '../constants';
+
+const NoteCreate = ({
+  history: { goBack },
+  location: { state },
+}) => {
+  const referredRecordData = state
+    ? {
+      name: state.entityName,
+      type: state.entityType,
+      id: state.entityId,
+    }
+    : null;
+
+  if (!state) {
+    return <Redirect to={ORDERS_ROUTE} />;
+  }
+
+  return (
+    <NoteCreatePage
+      domain={ORDERS_DOMAIN}
+      entityTypeTranslationKeys={{ poLine: 'ui-orders.poLine' }}
+      navigateBack={goBack}
```

Comment:
`goBack`, really?

Reply:
`if (!state) { return <Redirect to={ORDERS_ROUTE} />; }`
see above)